libs-scala/gatling-utils/src/main/scala/io/gatling/app/RunLikeGatling.scala

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package io.gatling.app
+
+import java.io.File
+import java.nio.file.FileSystems
+
+import akka.actor.ActorSystem
+import com.daml.scalautil.Statement.discard
+import com.typesafe.scalalogging.StrictLogging
+import io.gatling.core.config.GatlingConfiguration
+import io.gatling.core.scenario.Simulation
+
+import scala.util.Try
+
+object RunLikeGatling extends StrictLogging {
```

Comment:
Maybe nicer to replace this with something a little more elegant e.g. `Runner`.

Reply:
`io.gatling.app.Runner` already exists, it is the standard runner. I am going to rename it to `CustomRunner`